myhpi/core/templates/core/minutes.html

```diff
@@ -2,12 +2,15 @@
 {% load wagtailmarkdown %}
 
 {% block content %}
-    <div class="row">
+    <div class="row minutes-container">
         {% with page.text|markdown as parsed_md %}
-        <div class="col-9">
+        <div class="print-shown minutes-title">
+        <h1>{{  page.title }}</h1>
+        </div>
+        <div class="col-9 minutes-text">
             {{ parsed_md.0 }}
         </div>
-        <div class="col-3">
+        <div class="col-3 minutes-meta">
             <h4>Date</h4>
             <p>{{ page.date }}</p>
             <h4>Moderator</h4>
```

Comment:
What do you think about recreating the column layout of the old minutes? They use up less space and look cleaner, imo
![grafik](https://user-images.githubusercontent.com/44174681/171865242-c41a1089-57e4-4ded-9ca4-2f460127a5d4.png)

Reply:
Yeah, haven't checked it out yet either. Would be fine to me to open another issue for it